src/init.cpp

```diff
@@ -973,6 +976,11 @@ bool AppInitParameterInteraction()
     else
         LogPrintf("Validating signatures for all blocks.\n");
 
+    nMinimumChainWork = UintToArith256(uint256S(GetArg("-minimumchainwork", chainparams.GetConsensus().nMinimumChainWork.GetHex())));
+    if (nMinimumChainWork != UintToArith256(chainparams.GetConsensus().nMinimumChainWork)) {
+        LogPrintf("Setting nMinimumChainWork=%s (%s default value)\n", nMinimumChainWork.GetHex(), (nMinimumChainWork < UintToArith256(chainparams.GetConsensus().nMinimumChainWork)) ? "below" : "above");
```

Comment:
Regarding the stronger warning: it's hard to exactly say what may happen if nMinimumChainWork is too low, as it's just an anti-DoS threshold.  There's no reason to think that we would then sync to something that's not the most-work blockchain, unless there were some other attack going on.

I decided that since this is already a hidden option it's sufficient to just log a warning if the value is below the default, without specifying the risk.